web/frontend/src/components/camera/camera.vue

```diff
@@ -87,7 +89,7 @@ const viewFrame = async (cameraName: string) => {
     return;
   }
 
-  imgEl.setAttribute('src', URL.createObjectURL(blob));
+  imgEl?.setAttribute('src', URL.createObjectURL(blob));
```

Comment:
Upgrading Vue changed the type of HTML `ref`s. They now (accurately) report that they can possibly be undefined.